### PR TITLE
feat(`tests`): add maple as integration test

### PR DESCRIPTION
**Pull request by Evalir**

### CODE REVIEW

**Review by mattsse — DISMISSED** [review]

**Comment by mattsse** [comment]
> test / external integration tests / forking (pull_request) Successful in 59m

oh this runs for an hour ...

we should move this to another job and maybe only run on merge or once a day and open issue if fails, because this runs forever on PRs

**Review by mattsse — CHANGES_REQUESTED** [review]
this repo is huge and will take forever, so we need a new ci job for this think

File: crates/cli/tests/it/main.rs [hunk 1]
```diff
@@ -30,6 +30,9 @@ pub(crate) mod forge_utils;
 #[cfg(feature = "external-integration-tests")]
 mod integration;
 
+#[cfg(feature = "maple-integration-test")]
```
Comment (mattsse) [inline crates/cli/tests/it/main.rs:33]: maybe we want more, so we should rename this

**Review by mattsse — CHANGES_REQUESTED** [review]
do we get an issue if this fails?

File: crates/cli/tests/it/main.rs (hunk 1 above)
Comment (Evalir) [reply crates/cli/tests/it/main.rs]: done on https://github.com/foundry-rs/foundry/pull/5611/commits/b4c7dc10fe4895d0884a76de65859bf1ae8efaec & https://github.com/foundry-rs/foundry/pull/5611/commits/fb6a87e074cf193483ab2dbd3727706f95045c1e

**Comment by Evalir** [comment]
yep: see https://github.com/foundry-rs/foundry/pull/5611/commits/89d60120c9ca1d93267108d201ec803a608867a4

**Review by mattsse — APPROVED** [review]

**Pull request merged**